src/ggrc/assets/javascripts/components/inline_edit/inline.js

```
@@ -92,9 +93,11 @@
         value = instance.attr(property);
       }
       scope.attr('context.value', value);
-      if (_.isString(scope.attr('values'))) {
-        scope.attr('context.values', scope.attr('values').split(','));
+
+      if (values && _.isString(values)) {
```

Is checking for the truthiness of `values` needed here? Do we care if end up with `['']`?
BTW, would it perhaps be better to raise an error if an empty values list is provided to the component?